core/src/main/java/com/graphhopper/debatty/java/stringsimilarity/JaroWinkler.java

```diff
@@ -0,0 +1,144 @@
+package com.graphhopper.debatty.java.stringsimilarity;
+
+import java.util.Arrays;
+
+/**
+ * This class is copied from: https://github.com/tdebatty/java-string-similarity/blob/master/src/main/java/info/debatty/java/stringsimilarity/JaroWinkler.java
+ * and slightly modified. *
```

Comment:
Please [add the license](https://github.com/tdebatty/java-string-similarity/blob/master/LICENSE.md) directly to this file, similar to the Apache license headering.